index.html

```diff
@@ -568,7 +569,12 @@ <h4>Roles Supporting Name from Encapsulation</h4>
                 <h4>Roles Supporting Name from Legend</h4>
                 <div id="index_fromlegend">
                 </div>
-			</section>						
+			</section>
+            <section id="namefromheading">
+                <h4>Roles Supporting Name from Heading</h4>
+                <div id="index_fromheading">
```

Comment:
should `region` be in this list too?

Reply:
@scottaohara 

It is a good idea, but I am not sure how easy it would be for browsers to  implement (e.g. making an exception for `section` element) or for authors to understand how to use (e.g. not understanding they cannot use the technique when using the `section` element.   So need more discussion on this potential addition.  If you make it to easy for authors to add region landmarks, it also may contribute to landmark pollution on pages.

Reply:
right, i wasn't advocating for making it easy to add `region` landmarks.  I want to continue to avoid that.  

what i was advocating for is if an element is specifically given a `role=region` role, then the inheriting name from heading rules would make sense.

The rules for authors providing an explicit name to `section` elements to promote it to a `role=region` would still apply / not be changed.

Reply:
resolving this and will do another review in lieu of all the changes.